photocolle-sdk/photocolle-sdk.framework/README.mkd

```diff
@@ -1,31 +0,0 @@
-## INSTALLATION
```

Comment:
I see. You want to say that this PR convert from static framework to dynamic framework so revising  `README.mkd` should be done on another PR don't you?

If so, We do not remove `README.mkd`. Removing `README.mkd` is out of scope of this PR. I think.

But, in my opinion,  revising `README.mkd` is in scope of this PR. If a way of importing framework is changed according to this PR, description of that is modified at same time.

I'm also afraid to forgetting to revise `README.mkd`.

What do you think?